aws_eden_cli/cmdline.py

```diff
@@ -116,6 +123,91 @@ def command_config(args: dict):
         else:
             logger.info(f"Found {errors} errors")
 
+    elif args['push']:
+        dynamodb = boto3.client('dynamodb')
+
+        table_name = "eden"
```

Comment:
Shouldn't we hardcode a table name here? It can be passed with command options as described in the `baikonur-oss/terraform-aws-lambda-eden-api` docs below:

> Warning: DynamoDB table for state management is created by aws-eden-cli. Make sure to run `eden config --push` with success at least once before terraform apply. Default table name is `eden`.

Ref: https://github.com/baikonur-oss/terraform-aws-lambda-eden-api/pull/3

Reply:
On one hand one `eden` table and API module instance is sufficient, and on the other users may have their naming conventions for resources. It will be easier to make this changeable, but safer to hardcode.

Reply:
Parametrized.

Reply:
Parametrized.